docs/docs/mdx/importing-and-using-components.md

```diff
@@ -73,6 +73,48 @@ Because the `<Message />` and `<Chart />` components were passed into the provid
   lessonTitle="Make React components globally available as shortcodes in MDX"
 />
 
+## Lazy-loading components
+
+When you use components in your `.mdx` files, Gatsby will bundle them into the main application bundle. This can cause performance problems.
+
+In the future, [`gatsby-plugin-mdx`](/packages/gatsby-plugin-mdx) will address this. In the meantime, it can be prudent to lazy-load very large dependencies. The following snippet provides an example for lazy-loading an imaginary `Thing` component:
+
+```jsx:title=src/components/Thing/LazyThing.js
```

Comment:
Silly thing, but I feel like the path being this long might be unecessary? can it be `src/components/LazyThing.js` just to make it a bit more readable?